CHANGELOG.md

```diff
@@ -103,6 +103,15 @@
 * [BUGFIX] Querier: Fix stddev+stdvar aggregations to treat Infinity consistently. #9844
 * [BUGFIX] Ingester: Chunks could have one unnecessary zero byte at the end. #9844
 * [BUGFIX] OTLP receiver: Preserve colons and combine multiple consecutive underscores into one when generating metric names in suffix adding mode (`-distributor.otel-metric-suffixes-enabled`). #10075
+* [BUGFIX] PromQL: Ignore native histograms in `clamp`, `clamp_max` and `clamp_min` functions. #10136
+* [BUGFIX] PromQL: Ignore native histograms in `max`, `min`, `stdvar`, `stddev` functions and instead return an info annotation. #10136
+* [BUGFIX] PromQL: Ignore native histograms when compared to float values with `==`, `!=`, `<`, `>`, `<=`, `>=` and instead return an info annotation. #10136
+* [BUGFIX] PromQL: Return an info annotation if the `quantile` function is used on a float series that does not have `le` label. #10136
```

Comment:
I'm confused, I don't see where this change is sorry?

Reply:
See https://github.com/grafana/mimir/pull/10136 which  includes https://github.com/prometheus/prometheus/pull/15245 .
That PR 15245 contained a bunch of stuff, I just split it up for readability.

Reply:
But that PR does not contain any changes to the quantile function?

Reply:
promql/engine.go:+3079 ?  This is exactly why I asked @beorn7 to review the changelog, maybe I misunderstood something?

Reply:
Ah, I see it now, yes, thank you.